<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Btn_Start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>1900</width>
     <height>870</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>여기를 클릭하여 주문을 시작하세요.</string>
   </property>
  </widget>
  <widget class="QLabel" name="Question_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>40</y>
     <width>500</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard SemiBold</family>
     <pointsize>48</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>환영합니다!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
